v0/destinations/rockerbox/data/RockerboxTrackConfig.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "sourceKeys": "userId",
+    "destKey": "customer_id",
+    "sourceFromGenericMap": true
+  },
+  {
+    "sourceKeys": "email",
+    "destKey": "email",
+    "sourceFromGenericMap": true
+  },
+  {
+    "sourceKeys": "phone",
+    "destKey": "phone",
+    "sourceFromGenericMap": true
+  },
+  {
+    "sourceKeys": "timestamp",
+    "destKey": "timestamp",
+    "metadata": {
+      "type": "secondTimestamp"
+    },
+    "sourceFromGenericMap": true
```

Comment:
https://help.rockerbox.com/docs/webhooks some fields are required, so make them required.

Reply:
@ItsSudip Had missed the comment. Addressed.